src/object/schema_manager.c

```diff
@@ -2282,6 +2283,7 @@ sm_user_specified_name (const char *name, char *buf, int buf_size)
   if (name == NULL || name[0] == '\0')
```

Comment:
it needs to assert for "buf is not null" and "buf_size > 0"

Reply:
Thank you. I will add it.